src/sentry/api/endpoints/project_ownership.py

```diff
@@ -20,11 +26,29 @@
 HIGHER_MAX_RAW_LENGTH = 200_000
 
 
-class ProjectOwnershipSerializer(serializers.Serializer):
-    raw = serializers.CharField(allow_blank=True)
-    fallthrough = serializers.BooleanField()
-    autoAssignment = serializers.CharField(allow_blank=False)
-    codeownersAutoSync = serializers.BooleanField(default=True)
+class ProjectOwnershipRequestSerializer(serializers.Serializer):
+    raw = serializers.CharField(
+        required=False,
+        allow_blank=True,
+        help_text="Raw input for ownership configuration. Check out the [Ownership Rules Documentation](/product/issues/ownership-rules/) to learn more.",
```

Comment:
I think this sounds a bit better
```suggestion
        help_text="Raw input for ownership configuration. See the [Ownership Rules Documentation](/product/issues/ownership-rules/) to learn more.",
```